controllers/operatorscaler_controller.go

```diff
@@ -47,19 +49,21 @@ const (
 )
 
 type ResourceMappingRecord struct {
-	CrdName    string
-	ApiVersion string
-	Kind       string
-	PkgNames   []string
+	CrdName       string
+	ApiVersion    string
+	Kind          string
+	PkgNames      []string
+	ReportMetrics bool
```

Comment:
I don't think it's good to piggyback on this, I believe the original intention was this record will be supplied to the operator, better to have this if it makes sense for all the records.

Reply:
Maybe in future, we want to report this metrics for other storage systems also. Thats why this should be a good place IMO.

Reply:
I removed it and switched to the fixed list.